azurerm/provider.go

```diff
@@ -122,6 +122,7 @@ func Provider() terraform.ResourceProvider {
 		},
 
 		ResourcesMap: map[string]*schema.Resource{
+			"azurerm_action_group":                            resourceArmActionGroup(),
```

Comment:
given this a resource within Azure Monitor this should be `azurerm_monitor_action_group` (we should probably update the naming for the `azurerm_metric_alertrule` and `azurerm_autoscale_setting` resources too, but we should fix this for new resources going forward)